modern/src/settings/ComputedAttributesPage.js

```diff
@@ -40,11 +42,15 @@ const ComputedAttributeView = ({ updateTimestamp, onMenuClick }) => {
       <TableBody>
         {items.map((item) => (
           <TableRow key={item.id}>
-            <TableCell className={classes.columnAction} padding="none">
-              <IconButton onClick={(event) => onMenuClick(event.currentTarget, item.id)}>
-                <MoreVertIcon />
-              </IconButton>
-            </TableCell>
+            {adminEnabled &&
+              (
```

Comment:
Do we need these brackets? Same question for other places as well.

Reply:
No. Removed.